config/rbac/role.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
Should not be the name of Role and Cluster role `nats-manager-role`?

Reply:
same as other comments regarding kustomize prefix below